src/parser/parser.yy

```diff
@@ -866,7 +866,7 @@ predicate_expression
     | KW_EXISTS L_PAREN expression R_PAREN {
         if ($3->kind() != Expression::Kind::kLabelAttribute && $3->kind() != Expression::Kind::kAttribute &&
             $3->kind() != Expression::Kind::kSubscript) {
-            throw nebula::GraphParser::syntax_error(@3, "The exists only accept LabelAttribe, Attribute and Subscript");
```

Comment:
Maybe only dev facing?

Reply:
It's actually user-oriented,which is obviously not user-friendly, but the changes are intended.